Library/Formula/sassc.rb

```diff
@@ -2,7 +2,7 @@
 
 class Sassc < Formula
   homepage "https://github.com/sass/sassc"
-  url "https://github.com/sass/sassc/archive/3.1.0.tar.gz"
+  url "https://github.com/sass/sassc/archive/3.2.0.tar.gz"
   sha1 "573224ad922b46ea1a568807ddcbc7de41a4254d"
```

Comment:
This needs to be updated in both PRs. as well. `sha256` would be particularly :tada:.